setup.py

```diff
@@ -28,6 +28,7 @@
   py_modules=["vercel_ai"],
   include_package_data=True,
   install_requires=[
+    "requests",
```

Comment:
We already have curl_cffi, there's no need to add another dependency that does the same thing.